examples/agents/simple_controllers.py

```diff
@@ -140,7 +182,7 @@ def __init__(self, environment, k_p=10, k_i=0.01, controller_no=0, reference_idx
         super().__init__(environment, k_p, controller_no, reference_idx)
         action_space = environment.action_space
         assert type(action_space) is Box and type(
-            environment.physical_system) is DcMotorSystem, 'No suitable action space for P Controller'
+            environment.physical_system) is DcMotorSystem, 'No suitable action space for PI Controller'
```

Comment:
The class here is a PI controller -> hence, it should be of general form for arbitrary control problems.
In contrast to the class naming, the class content is a spezialized FOC-controller for three-phase drives which is not consistent. This specialized class may inhert from the genreal PI controller class, but this is not the case here. 

Likewise the name labeling and mapping from controller to motor classes above is a wild mix and I cannot find that starting from general controller definitions more spezialized motor-typ related controllers are derived.